frontend/src/__tests__/cypress/cypress/e2e/modelServing/ServingRuntimeList.cy.ts

```diff
@@ -315,6 +315,15 @@ describe('Serving Runtime List', () => {
     modelServingSection.getModelMeshRow('ovms').findExpansion().should(be.collapsed);
     modelServingSection.getModelMeshRow('ovms').findExpandButton().click();
     modelServingSection.getModelMeshRow('ovms').findExpansion().should(be.expanded);
+
```

Comment:
Add coverage for these cases:

1. Accelerator is disabled, inferenceservice contains accelerator support
2. Accelerator is enabled, inferencesrvice contains accelerator support
3. Accelerator is enabled, inferenceservice does not contain accelerator support